src/components/StockDetails/StockPredictionOverview.jsx

```diff
@@ -19,10 +19,10 @@ const CustomBox = ({children, ...otherProps}) => (
 const PredictionTable = () => (
     <CustomGridItem colSpan={2} shadow="none">
         <VStack spacing={4}>
-            <CustomBox>
+            <CustomContainer>
```

Comment:
Lets not call it customcontainer since it uses a box component and not a container component

Reply:
the reason being we have a public component called CustomBox, and I don't want people to mistake it for that. This current component is a placeholder anyway.